### ISSUE TITLE
Feature: Ability to add addresses to an already polling chainsaw instance

### ISSUE THREAD

**Issue by rongierlach** [SpankChain/chainsaw#5]
Would be nice to be able to poll more contracts than I started with, without having to create a new instance.

**Comment by yogeshgo05** [comment]
I see , if I understand correctly you want to be able add new contracts/abis to watch after the instance is created .

**Comment by rongierlach** [comment]
yes, I suppose I could just push a new address onto the instance's `contractAddresses` attr?
